readme.md

```diff
@@ -115,6 +115,13 @@ Run `brew cask install suspicious-package` or [download manually](http://www.mot
 [![](screenshots/SuspiciousPackage.png)](http://www.mothersruin.com/software/SuspiciousPackage/)
 
 
+### [QLRest](https://github.com/cluther/qlrest)
+
+> Preview ReST (ReStructuredText) files
+
+Run `brew cask install qlrest` or [download manually](https://github.com/downloads/cluther/qlrest/QLRest.qlgenerator.zip)
+
```

Comment:
Needs a screenshot (see https://github.com/sindresorhus/quick-look-plugins/pull/53/files#r86662217), and should be put into the `More` section as it's not something everyone would want.